client/src/main/java/com/distkv/dst/asyncclient/example/DstAsyncUsageExample.java

```diff
@@ -76,39 +84,87 @@ public static void main(String[] args) throws ExecutionException, InterruptedExc
         } else {
-          System.out.println("The result of dstClient.sets().get(\"k1\") is: "
-                  + r.getValuesList());
+          System.out.println("Set get completed.");
         }
       });
       dictGetFuture.whenComplete((r, t) -> {
         if (t != null) {
           throw new IllegalStateException(t);
         } else {
-          System.out.println("The result of dstClient.dicts().get(\"dict1\") is: "
-                  + "\n" + r.getDict());
+          System.out.println("Dict get completed.");
+        }
+      });
+
+      StringProtocol.GetResponse strGet = strGetFuture.get();
+      ListProtocol.GetResponse listGet = listGetFuture.get();
+      SetProtocol.GetResponse setGet = setGetFuture.get();
+      DictProtocol.GetResponse dictGet = dictGetFuture.get();
+
+      System.out.println("The result of dstClient.strs().get(\"k1\") is: "
+            + strGet.getValue());
+      System.out.println("The result of dstClient.lists().get(\"k1\") is: "
+            + listGet.getValuesList());
+      System.out.println("The result of dstClient.sets().get(\"k1\") is: "
+            + setGet.getValuesList());
+      System.out.println("The result of dstClient.dicts().get(\"dict1\") is: "
+            + "\n" + dictGet.getDict());
+
+      // SortedList example.
+      CompletableFuture<SortedListProtocol.PutResponse> slistPutFuture =
+            dstClient.sortedLists().put("k1", slist);
+      CompletableFuture<SortedListProtocol.PutMemberResponse> slistPutMFuture =
+            dstClient.sortedLists().putMember("k1", new SortedListEntity("s",100));
+
+      slistPutFuture.whenComplete((r, t) -> {
+        if (t != null) {
+          throw new IllegalStateException(t);
+        } else {
+          System.out.println("SortedList put completed.");
+        }
+      });
+
+      // Make sure the "Put" request is done first.
+      slistPutFuture.get();
+
+      slistPutMFuture.whenComplete((r, t) -> {
+        if (t != null) {
+          throw new IllegalStateException(t);
+        } else {
+          System.out.println("SortedList putMember completed.");
+        }
+      });
+      slistPutMFuture.get();
+
+      CompletableFuture<SortedListProtocol.TopResponse> slistTopFuture =
+            dstClient.sortedLists().top("k1", 3);
+
+      slistTopFuture.whenComplete((r, t) -> {
+        if (t != null) {
+          throw new IllegalStateException(t);
+        } else {
+          System.out.println("SortedList top completed.");
         }
       });
 
-      strGetFuture.get();
-      listGetFuture.get();
-      setGetFuture.get();
-      dictGetFuture.get();
+      SortedListProtocol.TopResponse slistTop = slistTopFuture.get();
+      System.out.println("The result of dstClient.sortedLists().top(\"k1\") is: " +
+            "{ First: " + slistTop.getList(0).getMember() +
+            "; Second: " + slistTop.getList(1).getMember() +
+            "; Third: " + slistTop.getList(2).getMember() + "; }");
```

Comment:
It also should output the **score** value which name is getMember().

Reply:
I think its not neccessary, maybe user just wanna to know who the top three are.
And this output format should be specified by users, here is just a simple example.

Reply:
The original intention of the design should output the score.

Reply:
I agree with @tortuo that we should show our intention to users in example.